.changeset/funny-countries-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/world": patch
```

Comment:
```suggestion
"@latticexyz/world": minor
```

this is _technically_ breaking but kinda more of an internal API and don't expect many downstream users to be affected, so maybe at least make this `minor`?

Reply:
I agree!